web-components/src/components/text-layouts/LabeledDetail.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box, styled, SxProps, Theme } from '@mui/material';
+import { Label as UnstyledLabel } from '../label';
+
+const Root = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(2),
+}));
+
+const Label = styled(UnstyledLabel)(({ theme }) => ({
+  color: theme.palette.info.main,
+  fontSize: 12,
+  [theme.breakpoints.up('sm')]: {
+    fontSize: 14,
+  },
+}));
+
+/** Grey label over child elements */
+export const LabeledDetail: React.FC<{
+  label: string;
+  sx?: SxProps<Theme>;
+  styles?: {
+    label?: SxProps<Theme>;
+  };
```

Comment:
this is a pattern that makes the most sense to me, but i'm not sure about it. Basically the `sx` prop would always be passed to the root element, then any sub-styles would be handled via a `styles` object. 

The alternative would be to have something like `styles.root` - but it's harder to combine `sx` so I think it makes sense to do what i've done here